runtime/Cargo.toml

```diff
@@ -19,6 +19,7 @@ crypto.workspace = true
 platform.workspace = true
 ufmt.workspace = true
 zerocopy.workspace = true
+arrayvec = { version = "0.7.4", default-features = false }
```

Comment:
We should add this as a workspace dependency in the root Cargo.toml

Reply:
Done